lib/facilities/vha_facility.rb

```diff
@@ -76,10 +76,17 @@ def attribute_map
           'feedback' => { 'health' => method(:satisfaction_data) },
           'services' => health_services,
           'mobile' => 'Mobile',
-          'mapped_fields' => mapped_fields_list
+          'mapped_fields' => mapped_fields_list,
+          'mappers' => { 'classification' => method(:classification_mapping) }
```

Comment:
Instead of adding `mappers`, you should be able to update the value of `classification` above to use `classification_mapping`.